nikola/plugins/compile/rest/__init__.py

```diff
@@ -26,6 +26,7 @@
 
 """reStructuredText compiler for Nikola."""
 
+import logging
```

Comment:
I'd move this one line down for the imports to be ordered but that might just be me :woman_shrugging: :smile: